scripts/build-sync-wheels

```diff
@@ -11,6 +11,12 @@ import argparse
 from pprint import pprint
 
 
+if os.geteuid() == 0:
+    # tar has issues when resetting permissions and ends up
+    # causing very subtle reproducibility issues.
+    print("This script cannot be run as root.")
```

Comment:
Nitpicking: Would it make sense to output to `STDERR` instead of `STDOUT`? I like the error message otherwise, it's a lot friendlier than letting future us figure out what's going on again!

Reply:
Sure.